_posts/2022-10-08-arrow-parquet-encoding-part-2.md

```diff
@@ -230,29 +230,29 @@ The Parquet encoding of the example would be:
 
 Closing out support for nested types are *lists*, which contain a variable number of other values. For example, the following four documents each have a (nullable) field `a` containing a list of integers
 
-```json
-{                     <-- First record
-  "a": [1],           <-- top-level field a containing list of integers
+```python
+{                     # <-- First record
+  "a": [1],           # <-- top-level field a containing list of integers
 }
 ```
-```json
-{                     <-- "a" is not provided (is null)
+```python
+{                     # <-- "a" is not provided (is null)
 }
 ```
-```json
-{                     <-- "a" is non-null but empty
+```python
+{                     # <-- "a" is non-null but empty
   "a": []
 }
 ```
-```json
+```python
 {
-  "a": [null, 2],     <-- "a" has a null and non-null elements
+  "a": [null, 2],     # <-- "a" has a null and non-null elements
 }
 ```
 
 Documents of this format could be stored in this Arrow schema
 
-```text
+```python
```

Comment:
I think these should still be text?

Reply:
I actually thought they look slightly better when formatted as "python" (as there is some visual difference):

<img width="792" alt="Screen Shot 2022-10-17 at 5 08 12 PM" src="https://user-images.githubusercontent.com/490673/196283777-1e449987-4b63-442b-82c0-2548147cbe64.png">

I can revert it you prefer

Reply:
LGTM - just wondered if it was accidental :sweat_smile: